### PR TITLE
[lineage-18.1] Update

### PR DESCRIPTION
**Pull request by derfelot**
Hold...still need to test & more coming

### CODE REVIEW

File: BoardConfigPlatform.mk [hunk 1]
```diff
@@ -70,6 +70,9 @@ BOARD_CACHEIMAGE_FILE_SYSTEM_TYPE := ext4
 BOARD_ROOT_EXTRA_FOLDERS := ocm
 TARGET_USERIMAGES_USE_EXT4 := true
 
+### POWER
+TARGET_USE_CUSTOM_POWERHINT ?= FALSE
```
Comment (derfelot) [inline BoardConfigPlatform.mk]: The `powerhint.json` is currently not included in a poplar build (lilac is probably fine, but haven't checked). Guessing the only reason is that this should be changed from `FALSE` to `false` to match the conditional. Will verify later today.

**Comment by derfelot** [comment]
WFD working fine now again.

**Comment by derfelot** [comment]
@Flamefire FYI

File: BoardConfigPlatform.mk (hunk 1 above)
Comment (Flamefire) [reply BoardConfigPlatform.mk]: @linckandrea I guess this should have really been `false`?

Comment (derfelot) [reply BoardConfigPlatform.mk]: fixed

**Pull request merged**